src/lib/Components/Home/ArtistRails/ArtistRail.tsx

```diff
@@ -161,7 +158,7 @@ class ArtistRail extends React.Component<Props, State> {
         </ScrollView>
       )
     } else {
-      return <Spinner style={{ flex: 1 }} />
+      return <Spinner style={{ flex: 1, marginBottom: 20 }} />
```

Comment:
I've had to add 20px here and there throughout code to balance layout - after heat is off would be good to sweep back over and clean up spinner CSS to make everything uniform

Reply:
/cc @mennenia